### DISCUSSION TITLE
快速入门 | OhMyLive2D

### DISCUSSION THREAD

**Discussion by giscus** [oh-my-live2d/oh-my-live2d#32] [Announcements]
# 快速入门 | OhMyLive2D

支持所有Live2D模型版本的Web组件, 快速为您的个人网站添加看板娘.

https://oml2d.com/guide/

**Comment by hacxy** [comment]
不错

**Comment by RoronoaZoro325** [comment]
厉害的，白嫖了哈哈哈，感谢！

**Reply by hacxy** [reply]
拒绝白嫖, 从我做起
求star 🥹

**Comment by ThisISMrzhang** [comment]
牛逼

**Comment by xingwangzhe** [comment]
star!

**Comment by wjt2018** [comment]
必须star！如果添加一个click的tips自定义功能就好了

**Comment by Almango** [comment]
加载失败

**Comment by csyh880a** [comment]
如果显示加载失败的可以用以下的的链接进行代替
https://m.a2b2.top/live2d/message.json

**Reply by csyh880a** [reply]
出了点bug，暂时不能用

**Comment by RedFlashR** [comment]
博主牛啊，star已给

**Comment by yuubaris** [comment]
牛哇！就是目前功能还是比较单一，真的是部署最简单的live2d了。
